tests/Fixer/CastNotation/CastSpacesFixerTest.php

```diff
@@ -21,18 +21,39 @@
  */
 final class CastSpacesFixerTest extends AbstractFixerTestCase
 {
+    public function testInvalidConfigMissingKey()
+    {
+        $this->setExpectedExceptionRegExp(
+            \PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException::class,
+            '#^\[cast_spaces\] Invalid configuration: The option "a" does not exist\. Defined options are: "space"\.$#'
+        );
+
+        $this->fixer->configure(['a' => 1]);
+    }
+
+    public function testInvalidConfigValue()
+    {
+        $this->setExpectedExceptionRegExp(
+            \PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException::class,
+            '#^\[cast_spaces\] Invalid configuration: The option "space" with value "double" is invalid\. Accepted values are: "none", "single"\.$#'
+        );
+
+        $this->fixer->configure(['space' => 'double']);
+    }
+
     /**
      * @param string      $expected
      * @param null|string $input
      *
-     * @dataProvider testFixCastsProvider
+     * @dataProvider provideSingleSpaceCases
      */
-    public function testFixCasts($expected, $input = null)
+    public function testFixCastsSingleSpace($expected, $input = null)
     {
+        $this->fixer->configure(['space' => 'single']);
```

Comment:
Please do not change this test by setting configuration. Add a new test that uses the same data provider instead. This way, tests prove that spaces are added with both default and `single` configuration.